common/changes/@itwin/presentation-common/schema-editing-replaceschemachangeswithdiffapi_2024-04-03-15-16.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/presentation-common",
+      "comment": "Removed deprecated calls of schemaItemTypeToString",
```

Change files should only mention changes that are important to package consumers. I don't think they care about us removing some deprecated calls.

```suggestion
      "comment": "",
```

---

I added a note to the nextversion.md file

---

That's great, but I think the change comment should still be removed.